src/Core/Abstractions/IStateService.cs

```diff
@@ -186,6 +186,8 @@ public interface IStateService
         Task<BwRegion?> GetActiveUserRegionAsync();
         Task<BwRegion?> GetPreAuthRegionAsync();
         Task SetPreAuthRegionAsync(BwRegion value);
+        Task<bool> GetShouldCheckOrganizationUnassignedItemsAsync(string userId = null);
```

Comment:
🤔  Naming wise, following a similar pattern to the other prompts (e.g. `Get/Set UnassignedItemsPromptShownAsync`) would make it easier to find this in the future.

Reply:
The problem is that this getter doesn't return if the prompt has been shown or not, it returns whether the check (and potentially the prompt) should be done or not, i.e. if the user chose "Remind me later" or not.
Also there are also two other that have `Should`:
- GetShouldTrustDeviceAsync
- GetShouldConnectToWatchAsync
Even though I agree having everything with the same name structure is better, but on the other hand I like having specific naming so one can exactly know what the method/property does without having to look much in other places.
What do you think?